src/content/en/fundamentals/primers/service-workers/the-players.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: shared/narrow
+published: false
+title: "The Players"
```

Comment:
Regarding this and the word 'control': I'm not ignoring you just delaying work while I think of a better idea.